site/content/docs/main/csi.md

```diff
@@ -1,20 +1,16 @@
 ---
-title: "Beta Container Storage Interface Snapshot Support in Velero"
+title: "Container Storage Interface Snapshot Support in Velero"
 layout: docs
 ---
 
 Integrating Container Storage Interface (CSI) snapshot support into Velero enables Velero to backup and restore CSI-backed volumes using the [Kubernetes CSI Snapshot APIs](https://kubernetes.io/docs/concepts/storage/volume-snapshots/).
 
 By supporting CSI snapshot APIs, Velero can support any volume provider that has a CSI driver, without requiring a Velero-specific plugin to be available. This page gives an overview of how to add support for CSI snapshots to Velero  through CSI plugins. For more information about specific components, see the [plugin repo](https://github.com/vmware-tanzu/velero-plugin-for-csi/).
 
```

Comment:
I thought GA was only for AWS/Azure? if that is the case, we should that clear.

Reply:
Would you suggest that I add the `Beta` back?

Reply:
I think its fine to take the beta out of the title. But if this is only GA for AWS and Azure, we make a note of that. Maybe something like:

```Note: Velero only considers the Velero Container Storage Interface (CSI) plugins [plugin version] GA if you are using them with the AWS or Azure provider plugins. If you are using the CSI plugins on a different cloud provider, the plugins are beta and it's not recommended that you use the CSI plugins on a production environment in this scenario. ```

i have a more limited understanding of the CSI plugin though, so this might not make sense in this context.

Reply:
Per discussion I'll move the NOTE regarding snapshot `durability` to the `pre-requisite` section.